### DISCUSSION TITLE
Brand Colors / Color Picker Icon

### DISCUSSION THREAD

**Discussion by darcimm** [twilio-labs/paste#1571] [Request: Icon] [answered]
Hi there!

I'd like to request two icons: "Brand Colors" and "Color Picker" for MC in SendGrid. As we're currently still using the SG design library, it may not make sense to add this to Paste quite yet but I wanted to begin a conversation.

I've provided initial ideas below, but I'm open to suggestions.

**Icon use case:** 
The 'Color Palette Sample' icon will be used to reference a new 'brand colors' feature which enables customers to save and re-use swatches at an account level. 
The 'Color Painting Palette' will be used to reference the generic color picker which is an existing feature.

![image](https://user-images.githubusercontent.com/83974127/122266902-e9a5a100-ce9f-11eb-95da-c1856a32f00f.png)

**Steamline icons:**
![image](https://user-images.githubusercontent.com/83974127/122267729-dcd57d00-cea0-11eb-88b7-f6b8dd4c9d23.png)

**Comment by kate-porter** [comment]
Hey @darcimm! Thanks for sharing these details—super helpful. Would you be able to share a screenshot of the UI that these icons will be used in?

**Reply by darcimm** [reply]
Hi @katieporter, sure - here are some screenshots from the editor and the design library where these icons will be used:

**Color Palette Sample (referencing new 'Brand Colors' feature)**
![image](https://user-images.githubusercontent.com/83974127/122815029-93b76b80-d29a-11eb-93a2-a45bc8d38b32.png)

![image](https://user-images.githubusercontent.com/83974127/122815086-a29e1e00-d29a-11eb-95d7-6d2f0bc3b309.png)

**Color Painting Palette (used to show/hide the color picker)**
![image](https://user-images.githubusercontent.com/83974127/122815224-c8c3be00-d29a-11eb-92c7-20eb4cffbb2b.png)

![image](https://user-images.githubusercontent.com/83974127/122815244-d11bf900-d29a-11eb-8cee-603dedc122ac.png)

**Reply by kate-porter** [reply]
Thanks for sharing these screenshots! I'm _super_ excited that brand colors are finally coming to MC. 🙌

I'll chat about these icons with the team and we'll get back to you shortly!

**Comment by kate-porter** [comment]
Hey @darcimm! Chatted with the team, and we've created tickets to add these icons in [Figma](https://issues.corp.twilio.com/browse/DSYS-2860) and in [Paste](https://issues.corp.twilio.com/browse/DSYS-2861).

I do have another question for you: we will likely need something conceptually similar to the "brand colors" icon for theming in our programmable apps, such as Flex - do you anticipate that the "brand colors" feature in MC will expand beyond colors, to include things like typography styles, in the future? If so, I'm wondering if we might be able to abstract that icon up a level to make it a "theme" icon rather than a "brand colors" icon. What do you think?

**Reply by darcimm** [reply]
Hi @katieporter thank you! We do anticipate adding text styles and logos to the 'Brand' tab in the Design Library in the future. We're thinking at some point users could be able to apply a defined 'brand' or 'theme' to a template in the editor to make the email design process quicker. 

I could see the _potential_ for this icon to serve as a 'theme' icon through which users can access brand colors, fonts, and logos, but this is not yet defined or prioritized. 

FYI, Here are other icons/images related to the design library currently in use -
Design Library (located in the main left nav):
![image](https://user-images.githubusercontent.com/83974127/123340255-8d7ef480-d511-11eb-81a3-3724ac38f87b.png)

Your Images (accessed via the code editor):
![image](https://user-images.githubusercontent.com/83974127/123340449-d9ca3480-d511-11eb-9cb5-7feb5423d5be.png)

Blank template (on the template page as an option to start from scratch):
![image](https://user-images.githubusercontent.com/83974127/123340380-b606ee80-d511-11eb-91fe-9aeb4fe3a49a.png)

**Reply by SiTaggart** [reply]
@katieporter any further thoughts on this?

**Reply by kate-porter** [reply]
@SiTaggart yes, we've got a design ticket for this in our sprint that just started (linked above in my previous comment)! Included in the AC on that ticket is finalizing the direction we want to take for the `brand-colors` vs. `theme` icon.

@darcimm if you think a `theme` icon will fit your needs, that's definitely something we'll work on getting added to Paste this sprint. It may very well look just like the Streamline icon you shared, but it would be named something like `icon-theme` and would likely be used in contexts outside of just MC in the future. Do you think having a `color-picker` icon and `theme` icon would fit your needs, or do you think you'll need a dedicated icon specifically for `brand-colors`?

**Reply by darcimm** [reply]
Hi @katieporter `theme` and `color-picker` will work for us for now. Thank you!

**Reply by kate-porter** [reply]
Awesome! We've got a ticket in our current sprint to get the icons finalized on the design side - I'll share a proposal with you sometime within the next couple of days. And then in the following sprint (starting July 14), we should get the icons added to Paste. We'll circle back with you once they are added to the system!

**Comment by andioneto** [comment, marked as answer]
Hi @darcimm 👋 Wanted to give an update that we have pulled a ticket into our current sprint to add the `color-picker` and `theme` icons. We will follow up with you after they are released. Expect it by mid next week. Thank you!

**Reply by gloriliale** [reply]
Hi @darcimm! The `color-picker` and `theme` icons have both been added to Paste! 🎉
